src/protagonist/API.Tests/Integration/CustomerQueueTests.cs

```diff
@@ -459,6 +460,7 @@ public async Task Get_BatchImages_200_IfImagesFound()
         var images = await response.ReadAsHydraResponseAsync<HydraCollection<DLCS.HydraModel.Image>>();
         images.TotalItems.Should().Be(3);
         images.Members.Should().HaveCount(3);
+        images.Members.All(x => x.Manifests.ContainsOnly("first")).Should().BeTrue();
```

Comment:
I don't think existing tests should be updated to check if `manifests` field is returned, unless it's relevant to the test. `Get_BatchImages_200_IfImagesFound()` is a focussed test that the correct counts are returned. Sticking assertion for `"manifests"` seems a bit unusual, ie there are no other fields like `"tags"` or `"roles"` being checked.